packages/frontend-2/pages/projects/[id]/index.vue

```diff
@@ -8,10 +8,19 @@
         @processed="onInviteAccepted"
       />
       <div
-        class="flex flex-col md:flex-row md:justify-between md:items-start gap-8 mb-6 mt-4 md:my-6"
+        class="flex flex-col md:flex-row md:justify-between md:items-center gap-4 my-2"
       >
         <ProjectPageHeader :project="project" />
-        <ProjectPageTeamBlock :project="project" class="w-full md:w-72 shrink-0" />
+        <div class="flex gap-x-4 items-center">
+          <div
+            class="text-body-3xs bg-foundation-2 text-foreground-2 rounded px-3 py-1 font-medium select-none whitespace-nowrap"
+          >
+            {{ project.modelCount.totalCount || 0 }} Project{{
```

Comment:
Shouldn't this be Model rather than Project

Reply:
Yup nice catch thanks :) Also added the versions counter in this branch, had to add a bunch of flex classes and wrapper but it was the only way, as all the alignments are quite different on mobile/desktop and with/without counter

<img width="1338" alt="Screenshot 2024-08-30 at 00 42 55" src="https://github.com/user-attachments/assets/ecb2e404-2889-4914-889b-35bd5a2f77ce">